environs/config/config.go

```diff
@@ -1000,6 +1019,28 @@ func (c *Config) EgressCidrs() []string {
 	return result
 }
 
+type FanConfigLine struct {
```

Comment:
I suggest moving this to the network package have a type called `FanConfig`.

There should be a function `ParseFanConfig(string) (FanConfig, error)` and the FanConfig should be able to serialize itself out appropriately for the various places it is needed.